test/Passes/pass-vectorization.mlir

```diff
@@ -0,0 +1,64 @@
+// RUN: tpp-opt %s --tile-consumer-and-fuse-producers -bufferize --vectorization-pass --split-input-file | FileCheck %s
```

Comment:
Tests should pull only what's necessary i.e., `-vectorization-pass` and canonicalization/cse if desired.
Could you update the test cases that they are in such shape the vectorization pass can consume directly?
Also a few negative test cases could be useful.

Reply:
Removed other passes, will add negative tests.